docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,597 @@
+- `FABRIC_CA_CLIENT_HOME` - Specify the path to where the generated certificates should be stored and which admin certificates should be used when users are registered and enrolled with a CA. This path is relative to the Fabric CA client binary.
+- `FABRIC_CA_CLIENT_TLS_CERTFILES` - Specify the location and name of the TLS certificate. If the path of the environment variable `FABRIC_CA_CLIENT_TLS_CERTFILES` is not an absolute path, it will be parsed as relative to the Fabric CA client’s home directory as specified by `FABRIC_CA_CLIENT_HOME`.
+- `FABRIC_CA_CLIENT_MSPDIR` - While you can use this environment variable to specify the name of the folder under `FABRIC_CA_CLIENT_HOME` where the generated certificates are stored, _a better option when you do not want the certificates to be stored in the default `msp` folder is to pass the `-M` flag on the enroll command itself and explicitly specify the location for the generated certificates._ If not specified on the command, the location defaults to `$FABRIC_CA_CLIENT_HOME/msp`. **Important:** When multiple members are enrolled with the CA, **you need to store their certificates in distinct folders** in order to differentiate them.  
+
+**Tip:** The first time you issue an `enroll` command from the CA client, if the `fabric-ca-client-config.yaml` does not already exist in the `$FABRIC_CA_CLIENT_HOME` directory, it is generated. When you customize the values in this file, they are used automatically by the CA client and do not have to be passed on the command line on a subsequent `enroll` command.
+
+Throughout these instructions, we modify the settings in the CA server configuration .yaml file and then override the settings with flags on the Fabric CA client CLI commands as needed. For example the register and enroll commands use the `--tls.certfiles` flag to specify the location of the TLS certificate. Because we have a single instance of the Fabric CA client binary that is used to interact with the three different CAs, (TLS CA, organization CA, intermediate CA), we use the `FABRIC_CA_CLIENT_HOME` environment variable to designate the location of the certificate folders for the targeted CA.
+
+## Submitting transactions from the CLI
+
+Two sets of CLI commands are included with the CA server and CA client binary files:
+
+- Use the [Fabric CA **server** CLI commands](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/servercli.html) to deploy and update the CA server.
+
+- Use the [Fabric CA **client** CLI commands ](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/clientcli.html) to submit requests to your CA server after it is set up, such as registering, enrolling or revoking identities.
+
+We will use both of these CLI commands throughout this topic.
+
+## What order should I deploy the CAs?
+
+Assuming you are not deploying a dual-headed CA that contains both a TLS CA and a root CA together, you would deploy the CAs in the following order:
```

Comment:
"both a TLS CA and a root CA"
Don't you mean "both the enrollment CA and TLS CA"?

Reply:
I decided on the term organization CA. If you prefer enrollment CA, let me know. 
Assuming you are not deploying a dual-headed CA that contains both a TLS CA and an organization CA together, you would deploy the CAs in the following order: